backend/stories/schemas.py

```diff
@@ -33,15 +38,22 @@ class CreateStory(BaseModel):
     current_location: str
     sickness_start: str
 
+
 class Story(CreateStory):
     id: int
+    token: str
+    user: User = None
     class Config:
         orm_mode: True
 
     @staticmethod
     def from_module(db_story: models.Story):
+        access_token = create_access_token(
+            data={"story_id": db_story.id}, expires_delta=timedelta(days=5)
```

Comment:
If the story was created by a logged user, we would be missing it's token information when creating this one.
We would have started with a request with a token with `email` but with `story_id: null` and we would be returning in the response a token with a `story_id` but with `email: null` right?

Reply:
We wouldn't have! The frontend dismisses story token information if the user was already authed... I think. *hurries off to check*